UI/window-basic-transform.cpp

```diff
@@ -267,6 +268,9 @@ void OBSBasicTransform::RefreshControls()
 	ui->cropRight->setValue(int(crop.right));
 	ui->cropTop->setValue(int(crop.top));
 	ui->cropBottom->setValue(int(crop.bottom));
+
+	oti_save = osi;
```

Comment:
Please use a more informative variable name here and use camelCase instead (we tend to prefer camelCase for C++ and ObjC and snake_case for pure C).